app/home/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+class Playlist(models.Model):
+    user_id = models.CharField(max_length=50)
+    playlist_id = models.CharField(max_length=50)
+    name = models.CharField(max_length=100)
+    is_master = models.CharField(max_length=50)
+
+class Group(models.Model):
+    group_id = models.CharField(max_length=50)
```

Comment:
`group_id` is really automatically generated. You get `id` for free (autoincremented postgres integer field). Probably only need the name here

Reply:
Maybe `owner = models.ForeignKey(User)` here

Reply:
maybe even `master_playlist = models.ForeignKey(Playlist)` here